packages/interbit-cli/scripts/deploy.js

```diff
@@ -74,6 +75,11 @@ const configureChains = async (cli, interbitManifest) => {
     }
 
     chainInterface.dispatch({ type: '@@interbit/DEPLOY' })
+
```

Comment:
@@interbit/DEPLOY may be redundant if start/deploy both have a setManifestAction. Leave for now.